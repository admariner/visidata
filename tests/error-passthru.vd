sheet	col	row	longname	input	keystrokes	comment
			open-file	sample_data/benchmark.csv	o	
benchmark	Quantity	1	select-equal-cell		,	
benchmark	Quantity		delete-cells		gzd	
benchmark	Quantity		addcol-expr	Quantity.foo	=	
benchmark	Quantity.foo		freq-col		F	
benchmark_Quantity.foo_freq	histogram		hide-col		-	Hide current column
