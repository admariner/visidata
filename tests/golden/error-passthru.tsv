Quantity.foo	count	percent
#ERR	51	100.00
